### DISCUSSION TITLE
AI4Green ELN

### DISCUSSION THREAD

**Discussion by JoeDavies-6** [TheELNConsortium/TheELNFileFormat#78] [General]
Hi all,

I'm Joe Davies, from Jonathan Hirst's research group at the University of Nottingham. We have been developing AI4Green, an open-source and free-to-use electronic laboratory notebook aimed at organic chemists, which integrates data-driven methods to encourage sustainable chemistry. 
https://ai4green.app/home 
https://github.com/AI4Green/AI4Green

We are interested in joining the ELN Consortium. In terms of the GitHub organisation, that is me and https://github.com/JoeHeeley2 . 

We have introduced a .ELN file export which successfully imports experiments into the eLab demo. We intend to add support for .ELN file imports in a future version.

**Comment by NicolasCARPi** [comment]
Hello,

Welcome :clinking_glasses: 

I've invited both accounts to the org.

Next step is to make a PR with an example .eln file so we can try and import it too, and comment on its structure and terms usage.

We also expect you to join us to our next meeting: https://github.com/TheELNConsortium/TheELNFileFormat/discussions/73

Make sure to Follow notifications of this repository so you can take part in the discussions.

**Reply by JoeDavies-6** [reply]
Thanks, will do!
